wordle.py

```diff
@@ -27,6 +58,8 @@ def invalidate(guess):
     if it does not have proper length or has non-alphabetic
     chars
```

Comment:
update this with logic-updated and return types.

Reply:
Updated